pretrain/scripts/v3-172b-exp2-sakura/sbatch.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#SBATCH --job-name=llama-2-172b-exp2
```

Comment:
slurmの管理上の都合で、実験番号をprefixしておきたいです。
```suggestion
#SBATCH --job-name=9_llama-2-172b-exp2
```